apps/src/applab/designElements/ColorPickerPropertyRow.jsx

```diff
@@ -28,7 +28,7 @@ var ColorPickerPropertyRow = React.createClass({
    * Make our button a colpick color picker, if it isn't already
    */
   ensureColorPicker: function () {
-    var element = React.findDOMNode(this.refs.colorPicker);
+    var element = ReactDOM.findDOMNode(this.refs.colorPicker);
```

Comment:
I see 12 instances of `findDOMNode(this.refs`.  That's easy enough to fix and it reads much better without the extra call.